Composer/packages/client/src/pages/home/ItemContainer.tsx

```diff
@@ -41,11 +50,12 @@ export const ItemContainer: React.FC<ItemContainerProps> = ({
       ref={forwardedRef}
       tabIndex={0}
       {...rest}
+      onKeyDown={handleKeyDown}
     >
```

Comment:
If we use a button here instead, then we'll get the onKeyDown behavior for free (because browsers using keyboard nav are smart enough to map "enter" onto a button click), and the screen-reader will read it as an interactive button. Right now it just calls it a "group" without saying it's interactable.

Reply:
done.